CI/scripts/fix_swig_wrap.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# #############################################################################
+# Simple script to fix SWIG-generated cc file to implement the fix suggested
+# by SWIG issue https://github.com/swig/swig/issues/1916
+#
+# Invoking Python wrappers to pass strings containing Unicode data
+# with surrogate characters runs into some conversion errors.
+#
+# The fix suggested by above issue is to replace:
+#    obj = PyUnicode_AsUTF8String(obj);
+# to obj = PyUnicode_AsEncodedString(obj, "utf-8", "surrogateescape");
+#
+# seems to workaround this issue. This script implements this string
+# replacement in the generated wrap*.cc file.
+
+# This script exists as other attempts to define SWIG-typemaps to implement
+# the workaound-fix in user-defined typemap()-ing code ran into other
+# compilation issues in the generated code.
+# This replacement is needed for these reasons:
+#
+#  - Even though the C++ variants of certain overloaded interfaces like
+#    init_client_ssl(), init_server_ssl() are ignored, the data for
+#    certain arguments [e.g., string &asn1_root_cert in these methods]
+#    contains Unicode surrogate chars. We need to apply this workaround-fix
+#    in order for the Python invocation to even succeed argument passing,
+#    even with the %ignore specifier in the SWIG.i interface file.
```

Comment:
@yelvmw -- Note: This point is new since the last time we visited this packet on Monday.

I have discovered thru unit-testing that for Python bindings of both the public `certifer_framework.h` and unit-testing `swigpytests.h`, we do need this shell script. 

Just using SWIG's `%ignore` directive alone to avoid overloaded interfaces is not sufficient to get all scenarios to work correctly.

Reply:
Would like to know a bit more of the details here. But if it's necessary, I won't block the pull request.

Reply:
@yelvmw -- I can talk to you offline to help clarify these gyrations.

Thru unit-testing, I found that we need both the `%ignore` directive in the SWIG-interface file -and- also the patch bash script to do the perl string replacement.